### PR TITLE
[16.0][MIG] partner_default_delivery_carrier: Migration to 16.0 

### PR DESCRIPTION
**Pull request by BT-mrobles**
issue https://github.com/OCA/delivery-carrier/issues/534

Migration from 14.0

### CODE REVIEW

**Comment by BT-mrobles** [comment]
@hildickethan-S73 @OscarIndias

**Review by hildickethan — APPROVED** [review]
code + functional review

**Review by BT-anieto — APPROVED** [review]

File: partner_default_delivery_carrier/models/res_config_settings.py [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+from odoo import fields, models
+
+
+class ResConfigSettings(models.TransientModel):
+    _inherit = "res.config.settings"
+
+    partner_default_delivery_carrier_id = fields.Many2one("delivery.carrier")
+
+    def get_values(self):
```
Comment (rousseldenis) [inline partner_default_delivery_carrier/models/res_config_settings.py:12]: This is not the good implementation in this version.

You should declare the field also on res.company. And here, change the field as related as : `related=company_id.partner_default_delivery_carrier_id`.

**Pull request closed without merging**